packages/app/pages/index.tsx

```diff
@@ -630,6 +646,9 @@ const CreateMembershipModal = ({
             <p>
               <b>Price:</b> {price} ETH
             </p>
+            <p>
+              <b>Royalty %:</b> {royaltyPcnt}%
```

Comment:
I was just keep the label as `Royalty`. Two `%` feels redundant

Reply:
done